Editor/API/Solver/PluginResolver.cs

```diff
@@ -96,14 +96,14 @@ public static IEnumerable<Type> FindPluginTypes() => AppDomain.CurrentDomain.Get
         private static IPluginInternal? InstantiatePlugin(Type pluginType) =>
             pluginType.GetConstructor(Type.EmptyTypes)?.Invoke(Array.Empty<object>()) as IPluginInternal;
 
-        public static IEnumerable<IPluginInternal?> FindAllPlugins() => FindPluginTypes().Select(InstantiatePlugin);
+        public static IEnumerable<IPluginInternal> FindAllPlugins() => FindPluginTypes().Select(InstantiatePlugin).Where(p => p != null)!;
```

Comment:
There should be a warning emitted if InstantiatePlugin returns null, citing the type in question (using unity's `Debug` facility). This would indicate that a plugin is registered with `[ExportsPlugin]` but does not have the right constructor type.